apps/style/applab/style.scss

```diff
@@ -459,11 +459,24 @@ button.arrow:focus {
   border-color: black;
   border-style: solid;
   border-width: 1px;
-  margin-top: 10px;
-  margin-bottom: 5px;
+  margin-top: 0px;
+  margin-bottom: 10px;
   font-size: 18px;
 }
 
+#designModeBox td {
+  vertical-align: top;
+}
+
+#divApplab * {
+  box-sizing: border-box;
```

Comment:
This seems like a big change. Are you sure we want to do this?

All elements will now be measured in a different (not standard) method (by including the border).

Also, everything now has a 2px margin, which will change the sizes of elements.

I'm a little concerned about canvas elements, the turtle image, etc.